azure.yaml

```diff
@@ -9,16 +9,17 @@ services:
     project: ./code
     language: py
     host: appservice
+    dist: ./dist
     hooks:
       prepackage:
         windows:
           shell: pwsh
-          run: poetry export -o requirements.txt;cd ./frontend;npm install;npm run build;
+          run: cd frontend;npm install;npm run build;cd ..;poetry export -o dist\requirements.txt;cp app.py dist;cp -r backend dist
           interactive: true
           continueOnError: false
         posix:
           shell: sh
-          run: poetry export -o requirements.txt;cd ./frontend;npm install;npm run build;
+          run: cd frontend;npm install;npm run build;cd ..;poetry export -o dist/requirements.txt;cp app.py dist;cp -r backend dist
```

Comment:
These are getting a bit long, thoughts on putting them in their own scripts?

Reply:
Now in a script